composer.lock

```diff
@@ -4,16 +4,21 @@
         "Read more about it at https://getcomposer.org/doc/01-basic-usage.md#installing-dependencies",
         "This file is @generated automatically"
     ],
-    "content-hash": "450c222b62c210972270203510876203",
+    "content-hash": "c7c535ea0423a04294bfcfa99144f8df",
     "packages": [
         {
             "name": "automattic/jetpack-abtest",
             "version": "dev-master",
+            "source": {
+                "type": "git",
+                "url": "https://github.com/Automattic/jetpack-abtest.git",
+                "reference": "26ab3b6142873fe8af01e542fdbeeef5e93364bb"
+            },
             "dist": {
-                "type": "path",
-                "url": "./packages/abtest",
-                "reference": "b873be98786907a49ac5cd21836167f662212aa0",
-                "shasum": null
+                "type": "zip",
```

Comment:
I'm fighting a bit with composer to get it to go back to using the symlink'd version.